snap/README.md

```diff
@@ -18,16 +18,22 @@ This directory contains the resources needed to build the Kata Containers
 $ sudo apt-get install -y snapd snapcraft
 ```
 
-## Build snap image
+## Install snap
+
+Kata Containers can be installed from the [snapcraft store][8] or by running next command
```

Comment:
Suggested rewrite to stay active: 
You can install Kata Containers from the [snapcraft store][8] or by running the following command: